extensions/3DTILES_extent/README.md

```diff
@@ -0,0 +1,136 @@
+# 3DTILES_extent
+
+## Contributors
+
+* Sean Lilley, [@lilleyse](https://github.com/lilleyse)
+* Samuel Vargas [@Samulus](https://github.com/Samulus)
+
+## Contents
+
+* [Overview](#overview)
+* [Defining Extents](#defining-extents)
+* [Holes in Extents](#holes-in-extents)
+* [Arc Types](#arc-types)
+* [Coordinate System](#coordinate-system)
+
+## Overview
+
+The extent 3dtiles extension allows the user to annotate the existence of a smaller 2D region(s) (an extent) in a given `tileset.json` via a 2D array of latitude, longitude pairs. This is useful for a variety of scenarios such as: Overlaying high resolution geometry ontop of low level geometry, insetting one tileset into another tileset, or clipping excess geometry inside of a provided tileset.
+
+## Defining Extents
+
+An extent is a collection of latitude and longitude coordinate pairs. Extents are two dimensional in nature, but an optional third component can be specified for **each** extent coordinate to specify its height (in meters). The coordinate pairs should be provided in **counterclockwise** winding order. Multiple extents can be specified. Convex and concave extents are both supported. At least three coordinates must be provided for an extent to be valid.
+
+The extent region definitions can be directly embedded in the `tileset.json` or located in a separate file and referred to using a `uri` reference in the corresponding `tileset.json` file.
+
+### External extent definition
+
+`tileset.json`
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "extensionsUsed": [
+    "3DTILES_extent"
+  ],
+  "extensions": {
+    "3DTILES_extent": {
+      "uri": "extent.json"
+    }
+  },
+  "geometricError": 100.0,
+  "root": {...}
+}
+```
+
+### `extent.json`
+
+```json
+{
+  "polygons": [
+    {
+      "coordinates": [
+        [32.511166344475825, -116.61512629247555],
+        [32.514456825587125, -116.61385130521423],
+        [32.515638616140173, -116.61829305963617],
+        [32.513154268152434, -116.61944030086889],
+        [32.511166344475825, -116.61512629247555]
+      ]
+    }
+  ]
+}
+```
+
+### Embedded extent definition
+
+`tileset.json`
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "extensionsUsed": [
+    "3DTILES_extent"
+  ],
+  "extensions": {
+    "3DTILES_extent": {
+      "extent": {
+        "polygons": [
+          {
+            "coordinates": [
+              [32.511166344475825, -116.61512629247555],
+              [32.514456825587125, -116.61385130521423],
+              [32.515638616140173, -116.61829305963617],
+              [32.513154268152434, -116.61944030086889],
+              [32.511166344475825, -116.61512629247555]
+            ]
+          }
+        ]
+      }
+    }
+  },
+  "geometricError": 100.0,
+  "root": {...}
+}
+```
+
+## Holes in Extents
+
+Holes are also supported, simpliy provide a `holes` object for a given polygon, e.g:

```

Comment:
Should probably be stated here as well that overlapping holes are treated as a boolean union.